### DISCUSSION TITLE
Responsible AI resources

### DISCUSSION THREAD

**Discussion by MatthewCalder-msft** [msusdev/Innovation-Challenge#2] [General]
Applying the Responsible AI standard is a key part of how judges will be evaluating projects. Here are a few good resources to get started:
[Empowering responsible AI](https://www.microsoft.com/en-us/ai/responsible-ai)
[Principles and approach](https://www.microsoft.com/en-us/ai/principles-and-approach/) 
[Tools and practices](https://www.microsoft.com/en-us/ai/tools-practices?culture=en-us&country=us)
[Responsible AI Toolbox](https://github.com/microsoft/responsible-ai-toolbox)
[What is Responsible AI](https://learn.microsoft.com/en-us/azure/machine-learning/concept-responsible-ai?view=azureml-api-2)
[Responsible AI dashboard](https://www.microsoft.com/en-us/ai/ai-lab-responsible-ai-dashboard)
[Assess AI systems by using the Responsible AI dashboard](https://learn.microsoft.com/en-us/azure/machine-learning/concept-responsible-ai-dashboard?view=azureml-api-2)

**Comment by MatthewCalder-msft** [comment]
Here is the link to the video [Responsible AI practices ](https://www.youtube.com/watch?v=xN06FGW8vvE)
And here is [the link where you can download the slide with all the links ](https://github.com/msusdev/Innovation-Challenge/blob/main/Responsible%20AI%20learning%20resources.pdf)to learning resources.

**Reply by Liza-the-Bookworm** [reply]
I used the link to the slide, but it is a pdf with no active links. Do we need any of those resources to turn in with our project?

**Reply by MatthewCalder-msft** [reply]
No you don't need to turn in any of that with your project. For your submission, you can just speak to how you applied the ideas of Responsible AI while working on your project. (To get at the links you need to download the pdf)

![image](https://github.com/msusdev/Innovation-Challenge/assets/48447848/b8fcc84c-e242-4940-9a4c-9e8300fca138)

**Comment by Liza-the-Bookworm** [comment]
Thank you!On Jun 10, 2024, at 3:41 PM, Matthew Calder ***@***.***> wrote:﻿
No you don't need to turn in any of that with your project. For your submission, you can just speak to how you applied the ideas of Responsible AI while working on your project. (To get at the links you need to download the pdf)
